.ebextensions/99datadog.config

```diff
@@ -0,0 +1,50 @@
+# .ebextensions/99datadog.config
+option_settings:
+    - namespace: aws:elasticbeanstalk:application:environment
+      option_name: DD_AGENT_HOST
+      value: "172.17.0.1"
+    - namespace: aws:elasticbeanstalk:application:environment
+      option_name: DD_AGENT_MAJOR_VERSION
+      value: "7"
+    - namespace: aws:elasticbeanstalk:application:environment
+      option_name: DD_AGENT_MINOR_VERSION
+      value: "" # Eg: 34 to install 7.34.x, leave empty to install the latest 7.x
+
+files:
+  "/etc/datadog-agent/datadog.yaml":
+    mode: "000640"
+    owner: root # will be changed to dd-agent after the installation
+    group: root
+    content: |
+      # Add here the Agent configuration
+      api_key: "@DD_API_KEY"
+      site: datadoghq.com
+      logs_enabled: true
+      apm_config:
+        enabled: "true"
+        apm_non_local_traffic: "true"
+      tags:
+        - service:postman
```

Comment:
`service:gogovsg`

Side note: No wonder I've been seeing strange traces in postman for the last couple of days lmao